src/services/AuthApi.ts

```diff
@@ -90,6 +96,10 @@ export default class AuthApi {
     );
   }
 
+  private static get verificationEmailRedirectUrl() {
+    return `${window.location.protocol}//${window.location.host}/account`;
```

Comment:
Don't need the protocol, prefix // means useing the same protocol as the website.

Reply:
This is good to know.

However, the user will be redirected to our app from Firebase so if we just prefix `//`, they will use the same protocol as Firebase website instead of our website. Although they're probably going to be HTTPS anyway but it's better to explicitly specify our protocol here.

Reply:
What do you mean? The function is invoked on our website right?

Reply:
Yea... When you say ysing the sane protocol as the website, which website do you mean?

Reply:
Same as the website that invokes the function.

Reply:
But how dows it know which website invoke the function? The function returns a plain string and we give Firebase that string... How does the browser know which protocol to use?

Reply:
You are coding too much lol. Scripts can only be run on your own website. You cannot run arbitrary scripts on another website.

Reply:
Yea but that's not what I mean...

So this url will be given to Firebase as a string. The verification process is doje in a firebase domain, not ours. After finish verifying the email, Firebase will redirect back to our website with this url. How does it know which protocol to use?